SPECS/kpatch/kpatch.spec

```diff
@@ -0,0 +1,74 @@
+Summary:        Kpatch tooling
+Name:           kpatch
+Version:        0.9.6
+Release:        1%{?dist}
+License:        GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Base
+URL:            https://github.com/dynup/kpatch
+Source0:        %{url}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+BuildRequires:  binutils
+BuildRequires:  dnf-utils
+BuildRequires:  elfutils
+BuildRequires:  elfutils-devel
+BuildRequires:  gcc
+BuildRequires:  glibc-devel
+BuildRequires:  kernel-headers
+
+%description
+Kpatch is a Linux dynamic kernel patching infrastructure which allows you to patch
+a running kernel without rebooting or restarting any processes.
+It enables sysadmins to apply critical security patches to the kernel immediately,
+without having to wait for long-running tasks to complete, for users to log off, or
+for scheduled reboot windows.
+It gives more control over uptime without sacrificing security or stability.
+
+%package devel
```

Comment:
Just making sure, since this is an unusual pattern- is there actually no dependency between the devel subpackage and the base package?

Reply:
Yeah, I wasn't sure what might be the best name for this subpackage, since on one hand you use it to build modules later used by `kpatch` but on the other hand there is no dependency on the default package from the `devel` one. Since it seems to immediately draw attention and questions, I'll rename it to `kpatch-build`. Thanks!

Reply:
New name sounds good, thanks!